Cargo.toml

```diff
@@ -1,8 +1,8 @@
 [package]
 name = "webgenr"
-version = "0.1.8"  # copy non-hbs files in templates (recursively)
+version = "0.1.9" 
```

Comment:
not a show stopper, but I find it helpful (for now at least) to include a comment about what is significant about this version

Reply:
good call.  

```suggestion
version = "0.1.9" # play audio in-line, add FontAwesome ▶️ for audio
```

Reply:
testing locally just in case Cargo chokes on the ▶️...